bower.json

```diff
@@ -1,19 +1,17 @@
 {
   "name": "bbc-vj-sharetools",
   "description": "BBC VJ Sharetools Module",
-  "main": "sharetools.js",
+  "main": "bin/",
   "authors": [
-    "fewstera <aidan.fewster@bbc.co.uk>"
+    "fewstera <aidan.fewster@bbc.co.uk>",
+    "ChrisBAshton"
   ],
   "license": "MIT",
   "homepage": "https://github.com/bbc/news-vj-sharetools",
   "moduleType": [],
   "ignore": [
-    "**/.*",
-    "node_modules",
-    "bower_components",
-    "test",
-    "tests"
+    "*",
+    "!bin/**"
   ],
-  "version": "1.0.8"
+  "version": "1.1.0"
```

Comment:
Ah, I updated this to `2.0.0` in package.json. Why do we have two places for our version numbers?